<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ep-cp-34" name="HealthProduct.HealthProductJourneyWithExcel" tests="17" failures="11" timestamp="5 Sep 2017 06:18:50 GMT" time="30.589" errors="6">
  <testcase name="CustomerEmail" time="5.208" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.CustomerEmail(HealthProductJourneyWithExcel.java:335)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- CustomerEmail -->
  <testcase name="QuotesBuyNow" time="0.043" classname="HealthProduct.HealthProductJourneyWithExcel">
    <error type="org.openqa.selenium.WebDriverException" message="Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 18 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;ep-cp-34&#039;, ip: &#039;192.168.3.245&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_144&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()=&#039;BUY NOW&#039;])[1]}">
      <![CDATA[org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 18 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()='BUY NOW'])[1]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at HealthProduct.HealthProductJourneyWithExcel.QuotesBuyNow(HealthProductJourneyWithExcel.java:391)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at <anonymous class>.nsCommandProcessor.prototype.execute(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12704)
	at <anonymous class>.Dispatcher.executeAs/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9558)
	at <anonymous class>.Resource.prototype.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9705)
	at <anonymous class>.Dispatcher.prototype.dispatch(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9652)
	at <anonymous class>.WebDriverServer/<.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:12517)
	at <anonymous class>.createHandlerFunc/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2054)
	at <anonymous class>.ServerHandler.prototype.handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2387)
	at <anonymous class>.Connection.prototype.process(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1223)
	at <anonymous class>.RequestReader.prototype._handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1677)
	at <anonymous class>.RequestReader.prototype._processBody(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1525)
	at <anonymous class>.RequestReader.prototype.onInputStreamReady(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1393)
]]>
    </error>
  </testcase> <!-- QuotesBuyNow -->
  <testcase name="GetLiveQuotes" time="0.034" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.GetLiveQuotes(HealthProductJourneyWithExcel.java:288)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- GetLiveQuotes -->
  <testcase name="Child" time="1.010" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.Child(HealthProductJourneyWithExcel.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- Child -->
  <testcase name="DOB" time="1.020" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.DOB(HealthProductJourneyWithExcel.java:213)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- DOB -->
  <testcase name="Adult" time="1.101" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.Adult(HealthProductJourneyWithExcel.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- Adult -->
  <testcase name="QuotesBuyNow" time="0.027" classname="HealthProduct.HealthProductJourneyWithExcel">
    <error type="org.openqa.selenium.WebDriverException" message="Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 15 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;ep-cp-34&#039;, ip: &#039;192.168.3.245&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_144&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()=&#039;BUY NOW&#039;])[1]}">
      <![CDATA[org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 15 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()='BUY NOW'])[1]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at HealthProduct.HealthProductJourneyWithExcel.QuotesBuyNow(HealthProductJourneyWithExcel.java:391)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at <anonymous class>.nsCommandProcessor.prototype.execute(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12704)
	at <anonymous class>.Dispatcher.executeAs/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9558)
	at <anonymous class>.Resource.prototype.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9705)
	at <anonymous class>.Dispatcher.prototype.dispatch(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9652)
	at <anonymous class>.WebDriverServer/<.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:12517)
	at <anonymous class>.createHandlerFunc/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2054)
	at <anonymous class>.ServerHandler.prototype.handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2387)
	at <anonymous class>.Connection.prototype.process(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1223)
	at <anonymous class>.RequestReader.prototype._handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1677)
	at <anonymous class>.RequestReader.prototype._processBody(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1525)
	at <anonymous class>.RequestReader.prototype.onInputStreamReady(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1393)
]]>
    </error>
  </testcase> <!-- QuotesBuyNow -->
  <testcase name="CustomerPopup" time="10.090" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.CustomerPopup(HealthProductJourneyWithExcel.java:312)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- CustomerPopup -->
  <testcase name="QuotesBuyNow" time="0.020" classname="HealthProduct.HealthProductJourneyWithExcel">
    <error type="org.openqa.selenium.WebDriverException" message="Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 13 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;ep-cp-34&#039;, ip: &#039;192.168.3.245&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_144&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()=&#039;BUY NOW&#039;])[1]}">
      <![CDATA[org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 13 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()='BUY NOW'])[1]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at HealthProduct.HealthProductJourneyWithExcel.QuotesBuyNow(HealthProductJourneyWithExcel.java:391)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at <anonymous class>.nsCommandProcessor.prototype.execute(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12704)
	at <anonymous class>.Dispatcher.executeAs/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9558)
	at <anonymous class>.Resource.prototype.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9705)
	at <anonymous class>.Dispatcher.prototype.dispatch(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9652)
	at <anonymous class>.WebDriverServer/<.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:12517)
	at <anonymous class>.createHandlerFunc/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2054)
	at <anonymous class>.ServerHandler.prototype.handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2387)
	at <anonymous class>.Connection.prototype.process(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1223)
	at <anonymous class>.RequestReader.prototype._handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1677)
	at <anonymous class>.RequestReader.prototype._processBody(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1525)
	at <anonymous class>.RequestReader.prototype.onInputStreamReady(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1393)
]]>
    </error>
  </testcase> <!-- QuotesBuyNow -->
  <testcase name="LaunchWebsite" time="3.529" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.LaunchWebsite(HealthProductJourneyWithExcel.java:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- LaunchWebsite -->
  <testcase name="EldestMember" time="1.109" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.EldestMember(HealthProductJourneyWithExcel.java:237)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- EldestMember -->
  <testcase name="FillLoginDetails" time="1.046" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.FillLoginDetails(HealthProductJourneyWithExcel.java:126)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- FillLoginDetails -->
  <testcase name="City" time="1.174" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.City(HealthProductJourneyWithExcel.java:264)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- City -->
  <testcase name="QuotesBuyNow" time="0.025" classname="HealthProduct.HealthProductJourneyWithExcel">
    <error type="org.openqa.selenium.WebDriverException" message="Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 16 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;ep-cp-34&#039;, ip: &#039;192.168.3.245&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_144&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()=&#039;BUY NOW&#039;])[1]}">
      <![CDATA[org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 16 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()='BUY NOW'])[1]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at HealthProduct.HealthProductJourneyWithExcel.QuotesBuyNow(HealthProductJourneyWithExcel.java:391)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at <anonymous class>.nsCommandProcessor.prototype.execute(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12704)
	at <anonymous class>.Dispatcher.executeAs/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9558)
	at <anonymous class>.Resource.prototype.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9705)
	at <anonymous class>.Dispatcher.prototype.dispatch(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9652)
	at <anonymous class>.WebDriverServer/<.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:12517)
	at <anonymous class>.createHandlerFunc/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2054)
	at <anonymous class>.ServerHandler.prototype.handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2387)
	at <anonymous class>.Connection.prototype.process(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1223)
	at <anonymous class>.RequestReader.prototype._handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1677)
	at <anonymous class>.RequestReader.prototype._processBody(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1525)
	at <anonymous class>.RequestReader.prototype.onInputStreamReady(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1393)
]]>
    </error>
  </testcase> <!-- QuotesBuyNow -->
  <testcase name="QuotesBuyNow" time="0.020" classname="HealthProduct.HealthProductJourneyWithExcel">
    <error type="org.openqa.selenium.WebDriverException" message="Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 11 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;ep-cp-34&#039;, ip: &#039;192.168.3.245&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_144&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()=&#039;BUY NOW&#039;])[1]}">
      <![CDATA[org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 11 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()='BUY NOW'])[1]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at HealthProduct.HealthProductJourneyWithExcel.QuotesBuyNow(HealthProductJourneyWithExcel.java:391)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at <anonymous class>.nsCommandProcessor.prototype.execute(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12704)
	at <anonymous class>.Dispatcher.executeAs/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9558)
	at <anonymous class>.Resource.prototype.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9705)
	at <anonymous class>.Dispatcher.prototype.dispatch(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9652)
	at <anonymous class>.WebDriverServer/<.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:12517)
	at <anonymous class>.createHandlerFunc/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2054)
	at <anonymous class>.ServerHandler.prototype.handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2387)
	at <anonymous class>.Connection.prototype.process(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1223)
	at <anonymous class>.RequestReader.prototype._handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1677)
	at <anonymous class>.RequestReader.prototype._processBody(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1525)
	at <anonymous class>.RequestReader.prototype.onInputStreamReady(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1393)
]]>
    </error>
  </testcase> <!-- QuotesBuyNow -->
  <testcase name="CustomerProceed" time="5.098" classname="HealthProduct.HealthProductJourneyWithExcel">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at HealthProduct.HealthProductJourneyWithExcel.CustomerProceed(HealthProductJourneyWithExcel.java:381)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- CustomerProceed -->
  <testcase name="QuotesBuyNow" time="0.035" classname="HealthProduct.HealthProductJourneyWithExcel">
    <error type="org.openqa.selenium.WebDriverException" message="Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 28 milliseconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;ep-cp-34&#039;, ip: &#039;192.168.3.245&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_144&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()=&#039;BUY NOW&#039;])[1]}">
      <![CDATA[org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Command duration or timeout: 28 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
*** Element info: {Using=xpath, value=(//span[text()='BUY NOW'])[1]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at HealthProduct.HealthProductJourneyWithExcel.QuotesBuyNow(HealthProductJourneyWithExcel.java:391)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.openqa.selenium.WebDriverException: Session not found: ccc18c83-070e-45e7-ada8-da23bcb7b0bd
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'ep-cp-34', ip: '192.168.3.245', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at <anonymous class>.nsCommandProcessor.prototype.execute(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12704)
	at <anonymous class>.Dispatcher.executeAs/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9558)
	at <anonymous class>.Resource.prototype.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9705)
	at <anonymous class>.Dispatcher.prototype.dispatch(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9652)
	at <anonymous class>.WebDriverServer/<.handle(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:12517)
	at <anonymous class>.createHandlerFunc/<(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2054)
	at <anonymous class>.ServerHandler.prototype.handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:2387)
	at <anonymous class>.Connection.prototype.process(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1223)
	at <anonymous class>.RequestReader.prototype._handleResponse(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1677)
	at <anonymous class>.RequestReader.prototype._processBody(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1525)
	at <anonymous class>.RequestReader.prototype.onInputStreamReady(file:///C:/Users/BHAWNA~1/AppData/Local/Temp/anonymous1429330007671885211webdriver-profile/extensions/fxdriver@googlecode.com/components/httpd.js:1393)
]]>
    </error>
  </testcase> <!-- QuotesBuyNow -->
</testsuite> <!-- HealthProduct.HealthProductJourneyWithExcel -->
